R/get_gambl_colours.R

```diff
@@ -0,0 +1,288 @@
+#' @title Get GAMBL Colours.
+#'
+#' @description Get GAMBL colour schemes for annotating figures.
+#'
+#' @details This function was designed to retrieve specified GAMBL colour palettes.
+#' By default, this function returns all the colours currently available.
+#' The user can easily specify what classification to return colors for with the `classification` parameter.
+#' It is also possible to return any given colour in different formats.
+#' To do so, refer to the Boolean arguments; `as_list` and `as_dataframe`.
+#' For more information regarding the available colours, refer to the utilities vignette.
+#'
+#' @param classification Optionally request only colours for pathology, lymphgen, mutation or copy_number.
+#' @param alpha Alpha of plotted colours.
+#' @param as_list Boolean parameter controlling the format of the return. Default is FALSE.
+#' @param as_dataframe Boolean parameter controlling the format of the return. Default is FALSE.
+#' @param return_available Set to TRUE for returning all available colours. Default is FALSE.
+#' @param verbose Default is FALSE
+#'
+#' @return A named vector of colour codes for lymphgen classes and pathology.
+#'
+#' @import dplyr ggsci stringr tidyr
+#' @export
+#'
+#' @examples
+#' lymphgen_cols = get_gambl_colours("lymphgen")
+#'
+#' \dontrun{
+#' #be sure to install ggsci from https://github.com/morinlab/ggsci
+#' #install_github("morinlab/ggsci")
+#' }
+#'
+get_gambl_colours = function(classification = "all",
+                             alpha = 1,
+                             as_list = FALSE,
+                             as_dataframe = FALSE,
+                             return_available = FALSE,
+                             verbose = FALSE){
+  
+  all_colours = list()
+  everything = c()
```

Comment:
The migration of this function does not add any new code, right? Just wanted to confirm, since everything is marked as "new". Thanks!

Reply:
Yes. it's exactly the same.